examples/websocket-basic-sample/websocket_basic_sample.bal

```diff
@@ -42,13 +49,20 @@ service<http:WebSocketService> basic bind { port: 9090 } {
         io:println("\nNew binary message received");
         io:print("UTF-8 decoded binary message: ");
         io:println(b);
-        caller->pushBinary(b) but { error e => log:printError("Error occurred when sending binary", err = e) };
+        caller->pushBinary(b) but {
+            error e => log:printError(
+                           "Error occurred when sending binary", err = e)
+        };
```

Comment:
This line also

Reply:
All BBE should only contain 80 characters in a given line. Since these are longer than 80 characters, examples are not shown in the expected way in BBE.
Please take a look at https://ballerina.io/learn/by-example/websocket-basic-sample.html